src/pyclass.rs

```diff
@@ -1,11 +1,11 @@
 //! `PyClass` and related traits.
+use crate::impl_::pyclass::PyClassItemsIter;
```

Comment:
Was this moved to a separate use clause intentionally?

Reply:
No, IDE auto import, will move into the clause. 👍